charts/hono/templates/hono-adapter-amqp/hono-adapter-amqp-vertx-deployment.yaml

```diff
@@ -51,6 +51,7 @@ spec:
               fieldPath: metadata.namespace
         {{- include "hono.component.springEnv" $args | indent 8 }}
         {{- include "hono.jaeger.clientConf" $args | indent 8 }}
+        {{- include "hono.component.envConfigMap" $args | indent 8 }}
```

Comment:
I guess `indent 6` needs to be used here.

Reply:
That's what you mentioned in your earlier comment as well. However, I do not think that's through. The aboven has something in the line of:
```
- name: MY_ENV_VAR
  value: my_value
```
the envconfigMap will be:
```
envFrom:
- configMapRef:
    name: my-config-map
```
The indent for both snippets, should be the same. The dash at the start of the line for the environment variables is at the same level as `envFrom`

Reply:
Right, I got that wrong.